test/hull_test.cpp

```diff
@@ -65,7 +65,7 @@ bool isMeshConvex(manifold::Manifold hullManifold, double epsilon = 0.0000001) {
 TEST(Hull, Tictac) {
   const double tictacRad = 100;
   const double tictacHeight = 500;
-  const int tictacSeg = 1000;
+  const int tictacSeg = 500;
```

Comment:
This didn't help the flaky missing one vertex issue - very weird.

Reply:
Yeah, I expected that this would work. I'll try to see if I can recreate it and find out why (also I do remember experimenting with this earlier as well at the start of the GSoC project, when I pointed out that the values were off by 1 or 2, we had discussed that it could be caused due to the precision errors, and it's possible that the algorithm itself isn't leading to the failure. Even then I was able to cause a missing vertex or two in both cases increasing and reducing the tictacSegs) https://github.com/opencax/GSoC/issues/89#issuecomment-2001985627

Reply:
Thanks - it'd be interesting first just to know which vertex is missing - I wonder if it's lined up with an axis or something? Once you know it's index, hopefully it won't be too hard to see where the algorithm is losing it.